pkg/operator/configobservation/apiserver/observe_apiserver.go

```diff
@@ -0,0 +1,274 @@
+package apiserver
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/imdario/mergo"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resourcesynccontroller"
+
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/configobservation"
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/operatorclient"
+)
+
+const (
+	userServingCertPublicCertFile            = "/etc/kubernetes/static-pod-resources/secrets/user-serving-cert/tls.crt"
+	userServingCertPrivateKeyFile            = "/etc/kubernetes/static-pod-resources/secrets/user-serving-cert/tls.key"
+	maxUserNamedCerts                        = 10
+	userServingCertResourceName              = "user-serving-cert"
+	namedUserServingCertResourceNameFormat   = "user-serving-cert-%03d"
```

Comment:
does this mean they end up as `user-serving-cert-4-7` in the operand namespace? revision 7.